airflow/utils/action_loggers.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+An Action Logger module. Singleton pattern has been applied into this module
+so that injected action loggers can be used all through the same python process.
+"""
+from __future__ import absolute_import
+import logging
+import airflow.settings
+
+
+class DefaultActionLogger:
```

Comment:
in py2* it's best practice to always derive `object` as in `class DefaultActionLogger(object):`, more information on the subject here: https://stackoverflow.com/questions/4015417/python-class-inherits-object

It'd be nice to get the linter to catch that.

Reply:
Good to know this. Will inherit from object.